### PR TITLE
Expose create team pop

### PR DESCRIPTION
**Pull request by pketh**
https://expose-create-team.glitch.me/
https://glitch.com/edit/#!/expose-create-team

updated new-team-pop adds random team names and error checking as you type

requires https://github.com/FogCreek/Glitch-Community/pull/98 and https://github.com/FogCreek/Glitch-Community/pull/84 to merge first

### CODE REVIEW

**Comment by pketh** [comment]
still wip, not ready for review yet

**Comment by pketh** [comment]
random name possibilities in https://glitch.com/edit/#!/expose-create-team?path=src/presenters/pop-overs/create-team-pop.jsx:24:1

**Comment by GregWeil** [comment]
When you create a new team the loader goes away for a moment before you get navigated to the new team. It should probably just stay around until the popover is closed by the navigation rather than flickering the input/button again.

File: src/presenters/pop-overs/user-options-and-create-team-pop.jsx [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+      backgroundColor: '',
+      hasCoverImage: false,
+      isVerified: false,
+    })
+    .then (response => {
+        this.setState({ isLoading: false });
+        window.location = `/@${response.data.url}`;
+      }).catch (() => {
+        this.setState({
+          isLoading: false,
```
Comment (pketh) [inline src/presenters/pop-overs/user-options-and-create-team-pop.jsx]: @GregWeil you should be able to fix https://github.com/FogCreek/Glitch-Community/pull/99#issuecomment-419553286 by removing 142. I'm gonna be stuck on editor stuff for a while, can you merge/release for me when you have time? Thx!

**Comment by GregWeil** [comment]
Fixes and cleanup, hidden behind the add-users dev toggle.

**Comment by GregWeil** [comment]
I don't think there are any blockers

**Comment by judeallred** [comment]
other than my comments above, lgtm to ship when you're ready

**Comment by GregWeil** [comment]
If a page is loaded with `#create-team` the create team pop will start open. It'll open even if you're an anon user, but not if you're logged out entirely (since the user popover won't exist, and there wouldn't be any users to put on the team)

**Comment by pketh** [comment]
if a user who has never visited glitch.com visits `#create-team` , what's expected to happen?

**Comment by GregWeil** [comment]
It'll load the page normally and not show the popover

**Comment by GregWeil** [comment]
(it'll open the popover now because a separate pull always creates anon users)

**Pull request merged**